frontend/lib/widgets/customInput.dart

```diff
@@ -0,0 +1,21 @@
+import 'package:flutter/material.dart';
+
+Container CustomInput({bool isPassword = false, TextEditingController controller, String hintText}){
```

Comment:
Pretty sure you can hot reload with function widgets. also here it wouldn't affect much this widget is only going to be used twice in the login. If we want the most optimized way we should use TextFormField widgets.

Reply:
I think if we're going to be making a customInput/TextField we should be making one that is going to be used everywhere, I'm scared that using a Container instead of a stateful widget textinput for CustomInput might cause the whole screen to rebuild with each change (This concern isn't certain).

Reply:
Nope it doesn't .